### ISSUE TITLE
Requeest on adding extensions to code sever

### ISSUE THREAD

**Issue by alfialdo** [coder/code-server#7144]
## What is your suggestion?

Can you add extensions like pylance and github co-pilot

https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance
https://code.visualstudio.com/docs/copilot/overview 

## Why do you want this feature?

Those tools are very helpful for developer, I've been using it for years the auto completion and quick code diagnostics is very

## Are there any workarounds to get this functionality today?

Can't find any so far

## Are you interested in submitting a PR for this?

-

**Comment by benz0li** [comment]
@alfialdo Regarding `Pylance`: Use `Jedi` instead.

See also https://github.com/coder/code-server/issues/6323.

**Comment by code-asher** [comment]
Yup, code-server uses Open VSX (https://open-vsx.org) so they would have to be added there, but it seems unlikely given their licenses.
